Sources/Models/ConfigurationMDB.swift

```diff
@@ -23,12 +23,12 @@ public struct ConfigurationMDB {
     let images = results["images"]
     base_url = images["base_url"].string
     secure_base_url = images["secure_base_url"].string
-    backdrop_sizes = images["backdrop_sizes"].arrayObject as! [String]
-    logo_sizes = images["logo_sizes"].arrayObject as! [String]
-    poster_sizes = images["poster_sizes"].arrayObject as! [String]
-    profile_sizes = images["profile_sizes"].arrayObject as! [String]
-    still_sizes = images["still_sizes"].arrayObject as! [String]
-    change_keys = results["change_keys"].arrayObject as! [String]
+    backdrop_sizes = images["backdrop_sizes"].arrayObject as? [String]
+    logo_sizes = images["logo_sizes"].arrayObject as? [String]
+    poster_sizes = images["poster_sizes"].arrayObject as? [String]
+    profile_sizes = images["profile_sizes"].arrayObject as? [String]
+    still_sizes = images["still_sizes"].arrayObject as? [String]
+    change_keys = results["change_keys"].arrayObject as? [String]
   }
```

Comment:
thanks for catching these 😓

Reply:
sounds like a great idea. Im not too sure of the negative side effects of the frameworks sharing the same name. Looking forward to your PR., let me know if you need a hand 😄

Reply:
Sure thing! In the next few days. 😊